### ISSUE TITLE
Batchnorm layers

### ISSUE THREAD

**Issue by b-biswas** [b-biswas/MADNESS#28]
As shown in this :[arXiv:1805.11604.pdf](https://arxiv.org/pdf/1805.11604.pdf)
perhaps adding some batch norm layers will make the latent space smooth?

**Comment by aboucaud** [comment]
I guess it never hurts :)
